ruby-training/trainees/shareni/test_katas/01/math_maddness/Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
```

Comment:
Why are you commiting code from @sbog8's directory? This may mean you didn't correctly pull from the master branch and merge into your own when she made changes. Please look into this.